Dockerfile-Windows

```diff
@@ -52,4 +52,4 @@ RUN mkdir -p /output/usr/bin && \
 FROM mcr.microsoft.com/windows/nanoserver:${OS_VERSION}
 COPY --from=velero-builder /output /
 
-USER ContainerUser
\ No newline at end of file
+USER ContainerUser
```

Comment:
Might want to remove this part, since there's no functional change to this file.

Reply:
okie.